modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -56,7 +61,19 @@ If you use SSH key-based authentication, you must manage the key with an SSH age
 # subscription-manager attach --pool=<pool_id>
 ----
 
-. Enable the repositories required by {product-title} {product-version}:
+. Enable the repositories required by {product-title} {product-version}.
+.. On a {op-system-base} 8 system:
++
+[source,terminal]
+----
+# subscription-manager repos \
+    --enable="rhel-8-for-x86_64-baseos-rpms" \
+    --enable="rhel-8-for-x86_64-appstream-rpms" \
+    --enable="ansible-2.9-for-rhel-8-x86_64-rpms" \
+    --enable="rhocp-4.10-for-rhel-8-x86_64-rpms"
+----
++
```

Comment:
Is the plus sign here necessary? I'd think it's not required for this nested OL, but I could be wrong.